topology/Indiana University/IU UITS/Condor collectors.yaml

```diff
@@ -1,52 +0,0 @@
-Disable: false
```

Comment:
This should be moved to where the collectors actually are (Nebraska?) and contact & support center info should be updated.

Reply:
yes, nebraska.

Reply:
Here? https://github.com/opensciencegrid/topology/tree/master/topology/University%20of%20Nebraska/Nebraska-Lincoln

Reply:
I think it should be the Omaha entry:
https://github.com/opensciencegrid/topology/tree/master/topology/University%20of%20Nebraska/Nebraska-Omaha